crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -407,31 +409,135 @@ fn entity_props_ui(
+    entity_props: &mut EntityProperties,
+) {
+    let query = ctx.current_query();
+
+    // Find closest pinhole transform, if any.
+    let mut pinhole_ent_path = None;
+    let mut cur_path = Some(entity_path.clone());
+    while let Some(path) = cur_path {
+        if let Some(re_log_types::Transform::Pinhole(_)) =
+            query_latest_single::<Transform>(&ctx.log_db.entity_db, &path, &query)
+        {
+            pinhole_ent_path = Some(path);
+            break;
+        }
+        cur_path = path.parent();
+    }
+
+    // Early out if there's no pinhole transform upwards in the tree.
+    let Some(pinhole_ent_path) = pinhole_ent_path else { return; };
+
+    entity_props.backproject_pinhole_ent_path = Some(pinhole_ent_path.clone());
+
+    let tensor = query_latest_single::<Tensor>(&ctx.log_db.entity_db, entity_path, &query);
+    if tensor.as_ref().map(|t| t.meaning) == Some(TensorDataMeaning::Depth) {
+        let tensor = tensor.as_ref().unwrap();
+
+        ui.checkbox(&mut entity_props.backproject_depth, "Backproject Depth")
+            .on_hover_text(
+                "If enabled, the depth texture will be backprojected into a point cloud rather \
+                than simply displayed as an image.",
+            );
+        ui.end_row();
+
+        if entity_props.backproject_depth {
+            ui.label("Pinhole");
+            ctx.entity_path_button(ui, None, &pinhole_ent_path)
+                .on_hover_text(
+                    "The entity path of the pinhole transform being used to do the backprojection.",
+                );
+            ui.end_row();
+
+            // Compute Auto scale values
+            let mut scale = *entity_props
+                .backproject_scale
+                .or(&re_data_store::EditableAutoValue::Auto(
+                    tensor.meter.map_or_else(
+                        || match &tensor.data {
+                            TensorData::U16(_) => 1.0 / u16::MAX as f32,
+                            _ => 1.0,
+                        },
+                        |meter| match &tensor.data {
+                            TensorData::U16(_) => 1.0 / meter * u16::MAX as f32,
+                            _ => meter,
+                        },
+                    ),
+                ))
+                .get();
+            let mut radius_scale = *entity_props
+                .backproject_radius_scale
+                .or(&re_data_store::EditableAutoValue::Auto(0.02))
+                .get();
+
+            ui.label("Backproject scale");
+            let speed = (scale * 0.05).at_least(0.01);
+            ui.add(
+                egui::DragValue::new(&mut scale)
+                    .clamp_range(0.0..=1.0e8)
+                    .speed(speed),
+            )
+            .on_hover_text("Scales the backprojected point cloud");
+            ui.end_row();
+
+            ui.label("Backproject radius scale");
+            let speed = (radius_scale * 0.05).at_least(0.01);
```

Comment:
When I tried it I found it a bit too sensitive still

Reply:
[out of scope] Also I noticed that units in points might be nice as well after all. That requires much deeper changes tho and I don't know how to make the ui story for that _ok_ - probably we need to make the setting thing we have on for point & line sizes on the space view "a thing" and re-use it here

Reply:
This feels pretty good with the new sensitivity I've just set